data-raw/pp.R

```diff
@@ -144,7 +144,7 @@ pp_tmax$PPORRES <- pp_tmax$TMAX
 PP <- bind_rows(pp_tmax, pp_npts, pp_lambda, pp_Ke, pp_cmax, pp_Clast, pp_AUC, pp_AUC_inf)
 
 # Constant variables
-PP$PPCAT <- "COMPARTMENTAL"
+PP$PPCAT <- unique(pc$PCTEST)
```

Comment:
after this update, `PPCAT` contains one value - `XANOMELINE` for all records - is this the intention?

```r
> count(pp, PPCAT)
       PPCAT    n
1 XANOMELINE 2352
```

Reply:
Still thinking about adding URINE records too

Reply:
I think it would be good to add some urine data too if at all possible please